scrapy/pqueues.py

```diff
@@ -63,21 +53,36 @@ def __init__(self, crawler, downstream_queue_cls, key, startprios=()):
         self.curprio = None
         self.init_prios(startprios)
 
+    @staticmethod
+    def _bring_prios_up_to_date(startprios):
+
+        def _is_valid(x):
+            return isinstance(x, (tuple, list)) and len(x) == 2
+
+        is_valid = all(map(_is_valid, startprios))
+        if is_valid:
+            return startprios
+
+        return [(x, None) for x in startprios]
+
     def init_prios(self, startprios):
         if not startprios:
             return
 
-        for priority in startprios:
-            self.queues[priority] = self.qfactory(priority)
+        startprios = self._bring_prios_up_to_date(startprios)
 
-        self.curprio = min(startprios)
+        for priority, state in startprios:
+            self.queues[priority] = self.qfactory(priority, state)
 
-    def qfactory(self, key):
+        self.curprio = min(p for p, s in startprios)
+
+    def qfactory(self, key, state):
         return create_instance(
             self.downstream_queue_cls,
             None,
             self.crawler,
             self.key + '/' + str(key),
+            state,
```

Comment:
This backward-incompatible change also needs to be covered in the release notes.

I wonder if it would not be better to deprecate the existing class and create a new one, with a new name (`PriorityQueue`?), with these changes, and let that new class become the default priority queue class.